doc/source/lilac/obtaining-building-and-running/obtaining-and-building-ctsm.rst

```diff
@@ -231,7 +231,7 @@ Example usage for a Mac (a simple case) is::
 
   ./lilac/build_ctsm ~/ctsm_build_dir --os Darwin --compiler gnu --netcdf-path /usr/local --esmf-mkfile-path /Users/sacks/ESMF/esmf8.0.0/lib/libO/Darwin.gfortranclang.64.mpich3.default/esmf.mk --max-mpitasks-per-node 4 --no-pnetcdf
 
-Example usage for NCAR's ``cheyenne`` machine (a more complex case) is::
+Example usage for NCAR's ``derecho`` machine (a more complex case) is::
 
   module purge
   module load ncarenv/1.3 python/3.7.9 cmake intel/19.1.1 esmf_libs mkl
```

Comment:
these modules are out of date and do not exist on derecho

Reply:
@billsacks same applies here.

Reply:
@billsacks I don't know if Jim's comments warrant opening an issue, but I will treat them as out of scope for this PR.